CONTRIBUTING.md

```diff
@@ -2,16 +2,16 @@
 
 1. Sign one of the contributor license agreements below.
 1. [Install Go](https://golang.org/doc/install).
-1. Get the package:
+1. Clone the repo:
 
-    `go get -d github.com/GoogleCloudPlatform/golang-samples`
+    `git clone https://github.com/GoogleCloudPlatform/golang-samples.git`
 1. Change into the checked out source:
 
-    `cd $(go env GOPATH)/src/github.com/GoogleCloudPlatform/golang-samples`
+    `cd golang-samples`
 1. Fork the repo.
 1. Set your fork as a remote:
 
-    `git remote add fork git@github.com:GITHUB_USERNAME/golang-samples.git`
+    `git remote add fork https://github.com/GITHUB_USERNAME/golang-samples.git`
 1. Make changes (see [Formatting](#formatting) and [Style](#style)), commit to
    your fork. Commit messages should follow the
```

Comment:
Would you mind updating this to point to Conventional Commits?

Reply:
Good idea, sure.